installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,228 @@
+
+include::modules/installation-aws-user-infra-bootstrap.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../support/troubleshooting/troubleshooting-installations.html#monitoring-installation-progress_troubleshooting-installations[Monitoring installation progress] for details about monitoring the installation, bootstrap, and control plane logs as an {product-title} installation progresses.
+
+* See xref:../../support/troubleshooting/troubleshooting-installations.adoc#gathering-bootstrap-diagnostic-data_troubleshooting-installations[Gathering bootstrap node diagnostic data] for information about troubleshooting issues related to the bootstrap process.
+
+* You can view details about the running instances that are created by using the link:https://console.aws.amazon.com/ec2[AWS EC2 console].
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-operators-config.adoc[leveloffset=+1]
+
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
+
+You can configure registry storage for user-provisioned infrastructure in AWS to deploy {product-title} to hidden regions. See xref:../../registry/configuring_registry_storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring the registry for AWS user-provisioned infrastructure] for more information.
+
+include::modules/registry-configuring-storage-aws-user-infra.adoc[leveloffset=+3]
+
+include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+
+include::modules/installation-aws-user-infra-delete-bootstrap.adoc[leveloffset=+1]
+
+include::modules/installation-create-ingress-dns-records.adoc[leveloffset=+1]
+
+include::modules/installation-aws-user-infra-installation.adoc[leveloffset=+1]
+
+include::modules/logging-in-by-using-the-web-console.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../web_console/web-console.adoc#web-console[Accessing the web console] for more details about accessing and understanding the {product-title} web console.
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service.
+
+[role="_additional-resources"]
+[id="installing-localzone-additional-resources"]
+== Additional resources
+
+* See link:https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacks.html[Working with stacks] in the AWS documentation for more information about AWS CloudFormation stacks.
+
+[id="installing-aws-localzone-next-steps"]
+== Next steps
```

Comment:
@kalexand-rh @GroceryBoyJr 
I am thinking if we need to provide a section, or reference, of deploying a simple app in Local Zone worker nodes, which require the user to define the tolerations on the deployment spec. I will check if we have something in our documentation to make the reference. Let me know if you consider it a requirement for now or we can patch it later.

Reply:
@mtulio, I think that we can patch that in after GA. Let me know if you have a reference doc, and I'll make an issue to track the work.

Reply:
x-ref: https://issues.redhat.com/browse/OSDOCS-4812